### DISCUSSION TITLE
Trade hook

### DISCUSSION THREAD

**Discussion by erictemponi** [azerothcore/azerothcore-wotlk#7623] [General]
Hey folks, how can I call a function when a player get on trade with another player? I took a look on ScriptMgr.h and there's no hook like OnPlayerTrade(...).  Maybe it is "OnPacketSend(WorldSession* /*session*/, WorldPacket& /*packet*/)", but I'm not sure about that.

**Comment by FrancescoBorzi** [comment]
this is more a question for StackOverflow:

https://www.azerothcore.org/wiki/how-to-ask-for-help
